Soomla/Assets/Plugins/Soomla/Levelup/Config/LevelUpSettings.cs

```diff
@@ -43,12 +43,15 @@ public class LevelUpSettings : ISoomlaSettings
 		static LevelUpSettings()
 		{
 			SoomlaEditorScript.addSettings(instance);
+
+			List<string> additionalDependFiles = new List<string>(); //Add files that not tracked in file_list
```

Comment:
@Blinkq, please remove this instantiation and put an empty array directly in method invocation. SOOMLA will split instantiation, if additional files will be added.
for example,
`additionalDependFiles.ToArray() => new string[] {}`